asciidoc_only/sample_chapter.asciidoc

```diff
@@ -106,10 +107,33 @@ Finally, here's a sample table:
 
 To insert a cross-reference, first add a block ID for the element to which you're referring. The block ID is a unique identifier enclosed in double square brackets. Place it on a single line above the block element (figure, table, section, etc.).
 
-Here's a cross-reference to <<unique_id_01>>. Here's one to <<unique_id_02>>. Finally, here's one to <<unique_chapter_id_01>>.
+Here's a cross-reference to <<example_code_block>>. Here's one to <<example_table>>. Finally, here's one to <<sample_chapter_unique_id>>.
 
 For the book to be valid, ids must be unique across the entire book, have no spaces, and not start with a number.
 
 O'Reilly house style calls for every formal figure, formal table, and formal code example to be preceded by a specific in-text reference. Formal figures and formal tables are created when you add captions to these elements.
 
 To include hyperlinks to external sources, add the full URL followed by the anchor text enclosed in single square brackets. This, for example, is a link to the https://www.oreilly.com/[O'Reilly home page]. The bracketed text will become a clickable link in web versions. In print versions, it will appear in the text, followed by the actual URL in parentheses.
+
+=== Creating a Book Build in Atlas 
+A helpful feature of Atlas is to create a book build in real time. This build shows the interior design of the book in PDF format, which allows you to see how elements will render on the print page. 
+
+First, go to the Dashboard tab of your book project. Then, look to the right panel, click the "PDF" box, and hit "Build". Atlas will take a moment to generate the pages. A "Download" button will appear - click this, and a PDF will open in your web browseer. This PDF may then be saved locally. 
+
+[TIP]
+====
+If the Build button on the configure tab is grayed out, simply refresh the browser. The Build button should then become active.
+====
+
+==== Adding a Chapter to the Book Build 
```

Comment:
Question: would it be worth noting that the same thing can be done in the _atlas.json_ file? Typically, I think authors who write in asciidoc/use atlas will be comfortable editing the json directly also, as an option.

Reply:
Good idea, added this in a Tip element